scripts/xtensa-build-all.sh

```diff
@@ -225,6 +225,16 @@ do
 		XTENSA_TOOLS_VERSION="RF-2016.4-linux"
 		HAVE_ROM='yes'
 	fi
+	if [ $j == "tgl" ]
+	then
+		PLATFORM="tigerlake"
+		ARCH="xtensa-smp"
+		XTENSA_CORE="X6H3CNL_2016_4_linux"
```

Comment:
As @ranj063 wrote core is `cavs2x_LX6HiFi3_2017_8` and then tools need to be from 2017.